src/file.ts

```diff
@@ -3477,6 +3490,25 @@ class File extends ServiceObject<File> {
   }
 }
 
+/**
+ * Basic Passthrough Stream that records the number of bytes read
+ * every time the cursor is moved.
+ */
+class ProgressStream extends Transform {
```

Comment:
Can this just be a PassThrough?

Reply:
isn't PassThrough extends from Transform?